migrations/2017_08_01_150952_create_audit_details_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateAuditDetailsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     * Creates database table for the audit_details
+     * @author Chandima Jayawickrama
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('audit_details',function(Blueprint $table){
+			$table->engine = 'InnoDB'->comment = "Primary key, auto increment";
```

Comment:
By default engine is "InnoDB" since MySql 5.5+ .  Also if you need to set engine, you need to do it in config file. And it is not primary key (as mentioned in your comment).